strix/data_checker.py

```diff
@@ -192,188 +91,40 @@ def get_train_valid_datasets():
     train_dataset, valid_dataset = get_train_valid_datasets()
     logger.info(f"Creating dataset '{cargs.data_list}' successfully!")
 
-    train_num = min(cargs.n_batch, len(train_dataset))
-    valid_num = min(cargs.n_batch, len(valid_dataset))
-    train_dataloader = DataLoader(train_dataset, num_workers=1, batch_size=train_num, shuffle=True)
-    valid_dataloader = DataLoader(valid_dataset, num_workers=1, batch_size=valid_num, shuffle=False)
-    train_data = first(train_dataloader)
-
-    img_key = cfg.get_key("IMAGE")
-    msk_key = cfg.get_key("MASK") if cargs.mask_key is None else cargs.mask_key
-    data_shape = train_data[img_key][0].shape
-    exist_mask = train_data.get(msk_key) is not None
-    channel, shape = data_shape[0], data_shape[1:]
-    logger.info(f"Data Channel: {channel}, Shape: {shape}")
+    if isinstance(train_dataset, torch.utils.data.DataLoader):
+        train_dataloader = train_dataset
+        valid_dataloader = valid_dataset
+    else:
+        train_num = min(cargs.n_batch, len(train_dataset))
+        valid_num = min(cargs.n_batch, len(valid_dataset))
+        train_dataloader = DataLoader(train_dataset, num_workers=1, batch_size=train_num, shuffle=True)
+        valid_dataloader = DataLoader(valid_dataset, num_workers=1, batch_size=valid_num, shuffle=False)
```

Comment:
Maybe we don't have to split train set, valid set and test set while checking data even, just check all the input data.

Reply:
Users may have different processing for training, validation, test dataset.